Sources/System/FilePath/FilePathString.swift

```diff
@@ -20,6 +20,30 @@ extension FilePath {
     self.init(_platformString: platformString)
   }
 
+  public init(platformString: [CInterop.PlatformChar]) {
+    guard let _ = platformString.firstIndex(of: 0) else {
+      preconditionFailure(
+        "input of FilePath.init(platformString:) must be null-terminated"
+      )
+    }
+    self = platformString.withUnsafeBufferPointer {
+      FilePath(_platformString: $0.baseAddress!)
+    }
+  }
+
+  public init(platformString: inout CInterop.PlatformChar) {
+    guard platformString == 0 else {
+      preconditionFailure(
+        "input of FilePath.init(platformString:) must be null-terminated"
+      )
+    }
+    self = withUnsafePointer(to: &platformString, FilePath.init(_platformString:))
```

Comment:
This just forces FilePath.init to build with an empty string. Is there a constant for an empty path?

Reply:
That's `FilePath()`, I guess.